app/design/frontend/kisilenko/kalexey_theme/Magento_Newsletter/templates/subscribe.phtml

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+// @codingStandardsIgnoreFile
+/** @var \Magento\Newsletter\Block\Subscribe $block */
+?>
+<div class="block newsletter">
+    <div class="title"><strong><?= $block->escapeHtml(__('Newsletter')) ?></strong></div>
```

Comment:
Cannot find, which lines were changed compared to the original file? Please specify the difference. It's good practice to commit the original overwritten file in one commit and change it in the next ones, but I cannot figure out where did you add your custom message?

Reply:
I was add into the wrong place. Now i put "Subscribe to our news" in the title